core/mondoo-azure-security.mql.yaml

```
@@ -1569,23 +1559,19 @@ queries:
         ```
 
         This script iterates over all PostgreSQL servers in the specified resource group, enforcing SSL on each one. Regularly running such scripts can help maintain compliance over time, especially in dynamic environments where new servers are frequently deployed.
-  - uid: mondoo-azure-security-ensure-that-ssl-enabled-postgresql-single
-    filters: |
-      asset.platform == "azure-postgresql-server"
-    mql: |
-      azure.subscription.postgreSql.server.properties.sslEnforcement == "Enabled"
-  - uid: mondoo-azure-security-ensure-that-ssl-enabled-postgresql-single-flexible
-    filters: |
-      asset.platform == "azure-postgresql-flexible-server"
-    mql: |
-      // TLS/SSL is enforced on the server by default.
-      azure.subscription.postgreSql.flexibleServer.configuration.where(name == 'require_secure_transport').all(value.downcase == "on")
   - uid: mondoo-azure-security-ensure-that-ssl-enabled-latest-version-mysql
     title: Ensure SSL connection enabled for MySQL Database Server with the latest version
     impact: 80
-    variants:
-      - uid: mondoo-azure-security-ensure-that-ssl-enabled-latest-version-mysql-single
-      - uid: mondoo-azure-security-ensure-that-ssl-enabled-latest-version-mysql-single-flexible
+    filters: |
+      asset.platform == "azure-mysql-flexible-server"
+    props:
+      - uid: latestTlsVersions
+        title: The latest supported TLS versions for MySQL flexible servers
+        mql: |
+          return ["TLSv1.2", "TLSv1.3", "TLSv1.2,TLSv1.3","TLSv1.3,TLSv1.2"]
```

```suggestion
          return ["TLSv1.2", "TLSv1.3", "TLSv1.2,TLSv1.3"]
```

It looks like we can revert this actually:

```
azure.subscription.mySql.flexibleServers.first.configuration.where: [
  0: {
    source: "system-default"
    dataType: "Set"
    description: "Which protocols the server permits for encrypted connections."
    defaultValue: "TLSv1.2,TLSv1.3"
    allowedValues: "TLSv1.2,TLSv1.3"
    value: "TLSv1.2,TLSv1.3"
    type: "Microsoft.DBforMySQL/flexibleServers/configurations"
    name: "tls_version"
  }
]```